core/src/main/java/io/questdb/griffin/engine/functions/bool/InSymbolFunctionFactory.java

```diff
@@ -79,7 +89,7 @@ public Function newInstance(int position, ObjList<Function> args, IntList argPos
         if (var.isConstant()) {
             return BooleanConstant.of(set.contains(var.getSymbol(null)));
```

Comment:
This may not work with bind variables

Reply:
We shouldn't get here unless the IN contents are a mix of constant literals and bind variables (e.g. `in (42, $1)`). That's because the `if` block checks `isConstant()` result and bind variable functions return `false` in this method. Am I missing anything?

Reply:
I think it's about case of

``` 
where cast('s' as symbol) in ($0)
```

`var` is the symbol column. I cannot think why it may be constant in real queries

Reply:
My bad, I misread the code. Fixed the (weird-looking) fast path in 41dc078a91eb00d5c058f6d902e883b66d350583